api/CacheStorage.json

```diff
@@ -7,8 +7,8 @@
           "chrome": {
             "version_added": "40",
             "notes": [
-              "Accessible from <code>Window</code> from version 43.",
-              "Accessible from <code>WorkerGlobalScope</code> from version 43."
+              "Accessible from <code>Window.caches</code> from version 43.",
```

Comment:
Note, I decided not to remove this from the top level version info - because otherwise the assumption might (on quick scan) that this can be used from version 40).  Should we remove from the methods though - e.g. `CacheStorage.match()`?